Userland/Applets/Keymap/KeymapStatusWindow.cpp

```diff
@@ -16,7 +16,7 @@ void KeymapStatusWidget::mousedown_event(GUI::MouseEvent& event)
     if (event.button() != GUI::MouseButton::Primary)
         return;
 
-    Core::Process::spawn("/bin/KeyboardSettings");
+    MUST(Core::Process::spawn("/bin/KeyboardSettings"));
```

Comment:
I've changed old calls of `Core::Process::spawn()` that discarded the return to `MUST()`, but I'm wondering if these should instead simply `(void)` the return?

Reply:
(I think they're all very unlikely to fail anyway)

Reply:
Unless spawning a separate process is vital for the program continuing (which seems unlikely?) I'd suggest something like:
```c++
auto spawn_result = Core::Process::spawn("/bin/KeyboardSettings");
if (spawn_result.is_error())
	GUI::MessageBox::show_error(window(), String::formatted("Failed to spawn KeyboardSettings: {}", spawn_result.error()));
```

Reply:
Do you think we could add something like `GUI::spawn_process_or_show_error()` to LibGUI, since this will be repeated quite a lot (across a load off apps).

So that'd be:
```c++
GUI::spawn_process_or_show_error("/bin/KeyboardSettings");
```

Reply:
Oooh, that sounds good!

Reply:
I've added this as `GUI::Process::spawn_or_show_error()`
Used like:
```c++
GUI::Process::spawn_or_show_error(window(), "/bin/KeyboardSettings");
```